stl/inc/vector

```diff
@@ -522,7 +522,16 @@ private:
 
     template <class _Iter>
     _CONSTEXPR20_CONTAINER void _Range_construct_or_tidy(_Iter _First, _Iter _Last, forward_iterator_tag) {
+#ifdef __cpp_lib_concepts
+        size_type _Count;
+        if constexpr (forward_iterator<_Iter>) {

```

Comment:
You already know that you have a forward_iterator

Reply:
I only know that I have either C++17 forward iterator or C++20 forward_iterator. Because as it turns out they are different things...

Reply:
Actually the test was wrong...